src/PowerShellEditorServices.Hosting/Commands/StartEditorServicesCommand.cs

```diff
@@ -39,33 +39,44 @@ public StartEditorServicesCommand()
             Environment.SetEnvironmentVariable("POWERSHELL_DISTRIBUTION_CHANNEL", "PSES");
             _disposableResources = new List<IDisposable>();
             _loggerUnsubscribers = new List<IDisposable>();
+
+            // These are the actual defaults. Because of how PowerShell cmdlets work, the "defaults"
+            // in the corresponding .PS1 file don't matter (though they match for discoverability),
+            // nor can these be set at the field declaration because the engine resets parameters to
+            // null on invocation.
```

Comment:
This was so annoying...thanks GitHub Copilot for the help because I was pounding sand wondering why the defaults were still null.